dev-lang/clojure/files/add-compile-spec-ant-build-target.patch

```diff
@@ -0,0 +1,54 @@
+From 3270c158d191f7e59540667c2dbe6c5dc7a19a59 Mon Sep 17 00:00:00 2001
+From: Florian Schmaus <flo@geekplace.eu>
+Date: Thu, 17 Jun 2021 14:52:01 +0200
+Subject: [PATCH] Add compile-spec ant build target
+MIME-Version: 1.0
+Content-Type: text/plain; charset=UTF-8
+Content-Transfer-Encoding: 8bit
+
+Suggested-by: Göktürk Yüksek <gokturk@gentoo.org>
+---
+ build.xml | 20 +++++++++++++++++++-
+ 1 file changed, 19 insertions(+), 1 deletion(-)
+
+diff --git a/build.xml b/build.xml
+index 4d29bbf9a27e..dc742113938f 100644
+--- a/build.xml
++++ b/build.xml
+@@ -85,6 +85,24 @@
+     </java>
+   </target>
+ 
++  <target name="compile-spec"
++          description="Compile Clojure spec sources."
++          depends="compile-java">
++    <java classname="clojure.lang.Compile"
++          classpath="${maven.compile.classpath}:${build}:${cljsrc}"
```

Comment:
Do you wanna leave `maven.compile.classpath` in classpath? You were afraid that it might cause some issues.